probe_scraper/runner.py

```diff
@@ -130,58 +130,54 @@ def load_moz_central_probes(cache_dir, out_dir):
     write_moz_central_probe_data(probes_by_channel, revisions, out_dir)
 
 
-def check_git_probe_structure(data):
+def check_glean_metric_structure(data):
     schema = Schema({
         str: {
-            And(str, lambda x: len(x) == 40): {
-                Optional("histogram"): [And(str, lambda x: os.path.exists(x))],
-                Optional("event"): [And(str, lambda x: os.path.exists(x))],
-                Optional("scalar"): [And(str, lambda x: os.path.exists(x))]
-            }
+            And(str, lambda x: len(x) == 40): [And(str, lambda x: os.path.exists(x))]
         }
     })
 
     schema.validate(data)
 
 
-def load_git_probes(cache_dir, out_dir, repositories_file, dry_run):
+def load_glean_metrics(cache_dir, out_dir, repositories_file, dry_run):
```

Comment:
I removed all interim objects that had to worry about `probe-type`, because we only have 1: `metric`. While this may seem confusing at first, I think in general it clears up what this code is trying to do.